doc/oci.rst

```diff
@@ -0,0 +1,140 @@
+Component Content in an OCI Registry
+====================================
+
+Component descriptors and local content delivered with a component are stored
+in an OCI registry as OCI artefact. The component descriptor may describe
+refernces to to other artefacts in any repository what type is supported
+by an access type, regardless of the type of the artefact. There are dedicated
+access types to refer to artefacts stored in an OCI registry.
+
+Storing Components
+------------------
+
+The content of a component is stored in the OCI registry as OCI artefact of the media type ``application/vnd.gardener.cloud.cnudie.component.config.v1+json``. Hereby the content consists at least of the component descriptor itself, which is basically stored as blob described as first layer in the oci manifest.
```

Comment:
Is the component descriptor blob now referenced by the digest or by the first layer entry?
So must the component descriptor blob be the first layer?

Reply:
we will always store is as first layer (before local artefacts), but is should be referenced via digest.